.github/workflows/check.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
         checking for unstated dependencies in vignettes .* NOTE
         checking top-level files .* NOTE
       unit-test-report-brand: >-
-        https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/thumbs/teal.png
+        https://raw.githubusercontent.com/insightsengineering/hex-stickers/thumbs/teal.png
```

Comment:
revert this change as well. link is broken

Reply:
```suggestion
        https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/thumbs/teal.png
```

Reply:
thanks for the suggestion @averissimo : D I was about to make a suggestion on my own